### ISSUE TITLE
Newtonsoft.Json.JsonReaderException

### ISSUE THREAD

**Issue by Goldy-the-fox** [roro08/Sketchfab-Ripper#22]
![image](https://user-images.githubusercontent.com/105525786/168402092-4b96a5fc-931a-4e19-8a24-ee551e687a50.png)
need to update, i think sketchfab has a new model encryption method or something similar...

**Comment by Tom5959** [comment]
there have answer

**Comment by trisadmeslek** [comment]
the existed answer is not a solution. it doesn't work

**Comment by Goldy-the-fox** [comment]
> the existing answer is not a solution. it doesn't work

yeah, i tested the modified configuration, and it still throws the error...

**Comment by KaWah1222** [comment]
> > 现有的答案不是解决方案。它不起作用
> 
> 是的，我测试了修改后的配置，它仍然抛出错误......

So do we have a solution?

**Comment by KaWah1222** [comment]
> > the existing answer is not a solution. it doesn't work
> 
> yeah, i tested the modified configuration, and it still throws the error...

So do we have a solution?

**Comment by moumouh206** [comment]
this is the new enryption key :
sfConfig=iqmKWuccsR1DWJLcPTT92OnLU1X94yCrmo5gswmXJ9Y7ADy3XoAfyXab4HFrkUs4yqxZb8ligH6eEMMZvVagj3UPESR0fhbNI4W6r/ezBVQuY86q5hsFUDpTc9UERZGZ
